api-reference/beta/toc.yml

```
@@ -11471,6 +11471,20 @@ items:
               href: api/unifiedRoleManagementPolicyRule-get.md
             - name: Update
               href: api/unifiedRoleManagementPolicyRule-update.md
+          - name: Resource namespace
+            href: resources/unifiedrbacresourcenamespace.md
+            items:
+            - name: List
+              href: api/unifiedrbacresourcenamespace-list.md
+            - name: Get
+              href: api/unifiedrbacresourcenamespace-get.md
+          - name: Resource action
+            href: resources/unifiedrbacresourceaction.md
+            items:
+            - name: List
+              href: api/unifiedrbacresourceaction-list.md
+            - name: Get
+              href: api/unifiedrbacresourceaction-get.md
```

Proposing to move Resource actions a level inside because it's dependent on Resource namespace
```suggestion
          - name: Resource namespace
            href: resources/unifiedrbacresourcenamespace.md
            items:
            - name: List
              href: api/unifiedrbacresourcenamespace-list.md
            - name: Get
              href: api/unifiedrbacresourcenamespace-get.md
            - name: Resource actions
              href: resources/unifiedrbacresourceaction.md
              items:
              - name: List
                href: api/unifiedrbacresourceaction-list.md
              - name: Get
                href: api/unifiedrbacresourceaction-get.md
```